CITATION.cff

```diff
@@ -3,8 +3,8 @@ cff-version: 1.2.0
 message: Please cite this crate using these information.
 
 # Version information.
-date-released: 2023-09-15
-version: 0.4.31
+date-released: 2024-01-22
```

Comment:
This commit duplicates https://github.com/chronotope/chrono/pull/1378?

Reply:
This should "wash out" when it's merged.

However, this can also be fixed by rebasing this PR branch to the updated `0.4.x` branch after PR #1378 is merged.